reference/3.0/Microsoft.PowerShell.Core/About/about_Redirection.md

```diff
@@ -21,130 +21,120 @@ You can use the following methods to redirect output:
 
 - Use the `Out-File` cmdlet, which sends command output to a text file.
   Typically, you use the `Out-File` cmdlet when you need to use its parameters,
-  such as the **Encoding**, **Force**, **Width**, or **NoClobber** parameters.
+  such as the `Encoding`, `Force`, `Width`, or `NoClobber` parameters.
 
-- Use the Tee-Object cmdlet, which sends command output to a text file and
+- Use the `Tee-Object` cmdlet, which sends command output to a text file and
   then sends it to the pipeline.
 
 - Use the PowerShell redirection operators.
 
-### POWERSHELL REDIRECTION OPERATORS
+### Powershell redirection operators
 
-The redirection operators enable you to send particular types of output to
-files and to the success output stream.
+The redirection operators enable you to send streams of data to a file or the
+**Success** output stream.
 
-The PowerShell redirection operators use the following characters to represent
-each output type:
+The PowerShell redirection operators use the following numbers to represent
+the available output streams:
 
-```
--   All output
-1   Success output
-2   Errors
-3   Warning messages
-4   Verbose output
-5   Debug messages
-```
+> [!NOTE]
+> There is also a **Progress** stream in PowerShell, but it is not used for
+> redirection.
+
+|Stream # |Description  |Introduced in |
+|---------|---------|---------|
+|1|**Success** Stream|PowerShell 2.0|
+|2|**Error** Stream|PowerShell 2.0|
+|3|**Warning** Stream|PowerShell 3.0|
+|4|**Verbose** Stream|PowerShell 3.0|
+|5|**Debug** Stream|PowerShell 3.0|
+|*|All Streams|PowerShell 3.0|
+
+The PowerShell redirection operators are as follows, where `n` represents
```

Comment:
Combine this into one paragraph.

The PowerShell redirection operators are as follows, where `n` represents 
the stream number. The **Success** stream ( `1` ) is the default if no stream is specified.